src/Tizen.NUI.Components/Controls/ScrollableBase.cs

```diff
@@ -281,20 +281,39 @@ public class ScrollEventArgs : EventArgs
         }
 
         /// <summary>
-        /// An event emitted when the scrolling starts, user can subscribe or unsubscribe to this event handler.<br />
+        /// An event emitted when the scrolling drag starts, user can subscribe or unsubscribe to this event handler.<br />
         /// </summary>
         /// <since_tizen> 6 </since_tizen>
         /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public event EventHandler<ScrollEventArgs> ScrollStartedEvent;
+        public event EventHandler<ScrollEventArgs> ScrollStartDragEvent;
```

Comment:
Do you think ScrollDragStartEvent and ScrollDragEndEvent as more consistent with ScrollAnimationStartEvent and ScrollAnimationEndEvent?

Note:  Even although called Drag they will still be emitted with Flick.   Add in documentation?

Reply:
Change ScrollStartDragEvent/ScrollEndDragEvent to ScrollDragStartEvent/ScrollDragEndEvent .